osf/migrations/0077_bannerimage_scheduledbanner.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2018-01-09 05:20
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import osf.utils.fields
+import osf.utils.storage
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0076_action_rename'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BannerImage',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('filename', models.TextField(unique=True)),
+                ('image', models.BinaryField()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ScheduledBanner',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('name', models.TextField()),
+                ('start_date', osf.utils.fields.NonNaiveDateTimeField()),
+                ('end_date', osf.utils.fields.NonNaiveDateTimeField()),
+                ('color', models.TextField()),
+                ('license', models.TextField(blank=True, null=True)),
+                ('default_photo', models.FileField(storage=osf.utils.storage.BannerImageStorage(), upload_to=b'')),
+                ('default_text', models.TextField()),
+                ('mobile_photo', models.FileField(storage=osf.utils.storage.BannerImageStorage(), upload_to=b'')),
+                ('mobile_text', models.TextField(blank=True, null=True)),
+            ],
+            options={
+                'permissions': (('view_banner', 'Can view banner details'), ('change_banner', 'Can change banner details'), ('delete_banner', 'Can delete banner')),
+            },
+        ),
+    ]
```

Comment:
In addition to this migration, can you add a migration to add the new permissions to the existing permission groups? I think the appropriate group names would be `read_only` and `osf_admin`